src/universalMiddleware/index.js

```diff
@@ -33,11 +33,12 @@ function universalReactAppMiddleware(request: $Request, response: $Response) {
     // API server, so we need to ensure it isn't firewalled, etc
     networkInterface: createNetworkInterface({
       uri: `http://localhost:${process.env.SERVER_PORT}/graphql`,
-      credentials: 'same-origin',
-      // transfer request headers to networkInterface so that they're accessible to proxy server
-      // Addresses this issue: https://github.com/matthew-andrews/isomorphic-fetch/issues/83
-      headers: request.headers,
-      ssrMode: true,
+      opts: {
+        credentials: 'same-origin',
```

Comment:
Updated the createNetworkInterface options to match the new requirements in 0.5.0